src/CSSList/CSSList.php

```diff
@@ -296,6 +296,21 @@ public function splice($iOffset, $iLength = null, $mReplacement = null)
         array_splice($this->aContents, $iOffset, $iLength, $mReplacement);
     }
 
+    /**
+     * Inserts an item before its sibling.
+     *
+     * @param RuleSet|CSSList|Import|Charset $oItem
+     * @param RuleSet|CSSList|Import|Charset $oSibling
+     */
+    public function insertBefore($oItem, $oSibling): void
+    {
+        if (in_array($oSibling, $this->aContents, true) === false) {
+            $this->append($oItem);
+        } else {
+            $this->replace($oSibling, [$oItem, $oSibling]);
```

Comment:
I think having the default case (the intended case) makes this method easier to understand. What do you think?

Reply:
I switched the two cases.